src/AppRouting.tsx

```diff
@@ -0,0 +1,41 @@
+import { RouterProvider, createHashRouter } from 'react-router-dom';
+import { PageNotFound } from './components/Pages/PageNotFound';
+import { App } from './App';
+import { HomePage } from './components/Pages/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { HomeRedirect } from './components/HomeRedirect';
+
+const router = createHashRouter([
+  {
+    path: '/',
+    element: <App />,
+    children: [
+      {
+        path: '/',
+        element: <HomePage />,
+      },
+      {
+        path: '/home',
+        element: <HomeRedirect />,
```

Comment:
you can use navigation component here
```suggestion
        element: <Navigate to="/" replace />,
```